lib/vsc/utils/generaloption.py

```diff
@@ -1058,7 +1060,16 @@ def configfile_parser_init(self, initenv=None):
                 the 2nd level key,value is the key=value. 
                 All section names, keys and values are converted to strings.
         """
-        self.configfile_parser = self.CONFIGFILE_PARSER()
+        # allow_no_value indicates that options can be unset (a.k.a. set to None) in configuration files,
+        # by specifying them without a value (for example: 'foo' rather than 'foo = bar')
+        # note: only available with Python 2.7
+        # see also https://docs.python.org/2/library/configparser.html#ConfigParser.RawConfigParser
+        if LooseVersion(sys.version) >= LooseVersion('2.7'):
+            self.log.debug("Initializing configfile parser, enable allowing to unset options")
+            self.configfile_parser = self.CONFIGFILE_PARSER(allow_no_value=True)
+        else:
+            self.log.debug("Initializing configfile parser, unsetting of options is not supported < Py2.7")
```

Comment:
behaviour won't be undefined, if you try to unset options using Python 2.6, you'll run into a `ParserError`, see unit tests

I'm happy to change this to 'warning' though